bin/alluxio-stop.sh

```diff
@@ -78,11 +75,54 @@ stop_proxies() {
 }
 
 stop_worker() {
+  if [[ ! -z "${cache}" ]] ; then
+    echo "Cache directory: ${cache}"
+    mkdir -p ${cache}
+    if [[ ${?} -ne 0 ]]; then
+      echo "Failed to create directory: ${cache}"
+      exit 2
+    fi
+
+    num_tiers=$(get_alluxio_property "alluxio.worker.tieredstore.levels")
+    for ((i=0;i<num_tiers;i++)); do
+      echo "Checking Worker tiered store level ${i}..."
+
+      tier_types=$(get_alluxio_property "alluxio.worker.tieredstore.level${i}.dirs.mediumtype")
+      tier_dirs=$(get_alluxio_property "alluxio.worker.tieredstore.level${i}.dirs.path")
+      # Use "Internal Field Separator (IFS)" variable to split strings on a
+      # delimeter and parse into an array
+      # - https://stackoverflow.com/a/918931
+      IFS=',' read -ra tier_types_arr <<< "${tier_types}"
+      IFS=',' read -ra tier_dirs_arr <<< "${tier_dirs}"
+
+      # iterate over the array elements using indices
+      # - https://stackoverflow.com/a/6723516
+      for j in "${!tier_types_arr[@]}"; do
+        tier_type=${tier_types_arr[$j]}
+        if [[ "${tier_type}" -eq "MEM" ]]; then
+          tier_dir=${tier_dirs_arr[$j]}
+
+          echo "Saving Worker tiered store at ${tier_dir} to ${cache}/tier${i}/${tier_dir}"
+          mkdir -p "${cache}/tier${i}/${tier_dir}"
+          cp -a "${tier_dir}/." "${cache}/tier${i}/${tier_dir}/"
```

Comment:
Have you considered the case where the ramdisk doesn't fit into in the directory/disk that we're copying to?

Reply:
The assumption from the ticket [AE-1416](https://tachyonnexus.atlassian.net/browse/AE-1416) is that there is sufficient disk space to hold the ramdisk.

Reply:
Thanks - let's assume that there is sufficient space in both cases (copy to/from). I'll resolve the other comment